src/Store/CookieStore.php

```diff
@@ -259,16 +285,20 @@ private function encrypt(
             throw \Auth0\SDK\Exception\ConfigurationException::requiresCookieSecret();
         }
 
+        // @codeCoverageIgnoreStart
         if ($ivLen === false) {
             return '';
         }
+        // @codeCoverageIgnoreEnd
 
         $iv = openssl_random_pseudo_bytes($ivLen);
 
+        // @codeCoverageIgnoreStart
         // @phpstan-ignore-next-line
         if ($iv === false) {
             return '';
         }
+        // @codeCoverageIgnoreEnd
 
         $encrypted = openssl_encrypt(serialize($data), self::VAL_CRYPTO_ALGO, $secret, 0, $iv, $tag);
         return base64_encode(json_encode(serialize(['tag' => base64_encode($tag), 'iv' => base64_encode($iv), 'data' => $encrypted]), JSON_THROW_ON_ERROR));
```

Comment:
This is too much =)

`base64_encode(json_encode(serialize()))`. It is enough to do `json_encode()`. We dont need to serialize an array with strings. `json_encode` is fine. 
We also dont need to run `base64_encode()` because a cookie can hold `[]{}"` and "base64 safe" characters. 

(If you change this, remember to make change the `decrypt()` method too)

Reply:
Good catch, I guess I wasn't giving json_encode() enough credit for handling the encrypted payload! Just tested, and it does work fine without it, and naturally leads to much smaller cookies.